src/Mhkb/ApiDoc/Commands/GenerateDocumentation.php

```diff
@@ -284,12 +284,32 @@ private function processLaravelRoutes(AbstractGenerator $generator, $allowedRout
         $withResponse = $this->option('noResponseCalls') === false;
         $routes = $this->getRoutes();
         $bindings = $this->getBindings();
+        //Per route binding
+        $route_bindings = config('api.bindings');
```

Comment:
@ivanhennig the variables throughout this repository are camelCase, can you change your newly added variables to be the same.